.github/workflows/ci.yml

```diff
@@ -83,7 +83,17 @@ jobs:
       - name: "Build"
         run: mvn -B -U install -DskipTests
       - name: "Create zip"
-        run: zip -r to-scan.zip engines/aio/target/alfresco-transform-core-aio-*.jar engines/base/target/alfresco-base-t-engine-*.jar model/target/alfresco-transform-model-*.jar
+        # Removing the aspectjweaver and bouncycastle jars from the scan, since Veracode detects them as 1st party code and fails the scan. TO BE REVERTED ONCE VERACODE FIXES THE ISSUE
```

Comment:
Let's move this comment down just above the actual shell line that does this: https://github.com/Alfresco/alfresco-transform-core/pull/956/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR95

Reply:
For some reason, adding a comment inside of run command was causing a syntax error, which I fixed by moving the comment to where it is now
https://github.com/Alfresco/alfresco-transform-core/actions/runs/9092835071/workflow
https://github.com/Alfresco/alfresco-transform-core/actions/runs/9092752774/workflow

Reply:
I reckon that's because it was formatted badly, the comment has to start at the same indentation level as the rest of the shell commands, not at the start of the line. This will mess up the YAML interpreter.

You can most definitely add comments in the Shell script that's run via `run: |` without any issues.